### DISCUSSION TITLE
Iris cooldown

### DISCUSSION THREAD

**Discussion by chshersh** [chshersh/iris#99] [General]
Hey everyone,

During the last Hacktoberfest, I had to review tons of PRs, answer and create multiple issues in many repositories I have so I got a burnout from the amount of work I needed to do 😫 

I still plan to continue my work on Iris at some point and prepare the v0.1.0.0 release but I don't know when will it happen. At the moment, there're a few open PRs in the repository that still require my review:

* https://github.com/chshersh/iris/pull/73
* https://github.com/chshersh/iris/pull/93
* https://github.com/chshersh/iris/pull/95
* https://github.com/chshersh/iris/pull/97
* https://github.com/chshersh/iris/pull/98

I'll provide my review when I'll be able to get back to this project. And when I do this, I'll ask whether you're still interested in pushing the PRs over the finish line. But of course, it's okay for you to not do any work further, it's OSS in the end 🙂  Appreciate your patience! 🙏🏻

Anyway, this is just a quick status update. Really appreciate everyone who contributed to my project so far! 🤗 

cc @initial-mockingbird @marcellourbani @lillycat332 @german1608

**Comment by marcellourbani** [comment]
I definitely will. Only paused working on this as looks like you have enough on your plate for now
Thanks for doing this

**Comment by german1608** [comment]
@chshersh don't worry at all. Take your time, not urgent at all. Just ping us when we need. And thanks so much for your effort, we know it's not easy!

**Comment by lillycham** [comment]
take as much time as you need 💜!

**Comment by chshersh** [comment]
Hey everyone 👋🏻 

I wanted to say big THANK YOU🙏🏻 for all your contributions and patience!
As of now, all outstanding PRs are merged 🎉 🚀 

I'm planning to prepare and publish `v0.1.0.0` release of Iris during February with all your awesome contributions included ✨ 

Thanks again!

**Comment by chshersh** [comment]
After a long time, I published the v0.1.0.0 release including all your awesome contributions 🤗 

* https://github.com/chshersh/iris/releases/tag/v0.1.0.0
